<?xml version="1.0"?>
<launch>
<!-- <include file="$(find jaco2_moveit_config)/launch/moveit_full_rviz.launch"/> -->

<node respawn="false" pkg="jaco2_simple_moveit_apps" type="move_to_new_rand_configuration" name="dyn_calib_sampling" output="screen">
  <param name="/jaco_use_accelerometer_calib" type="bool" value="true" />
  <param name="configuration_list" type="string" value="$(find jaco2_calibration_utils)/data/used_configurations.conf"/>
  <param name="save_path" type="string" value="$(find jaco2_calibration_utils)/data/used_configurations.conf"/>
  <param name="z_coord"   type="double" value="0.04"/>
  <param name="xm_coord"  type="double" value="-1"/>
  <param name="xp_coord"  type="double" value="1.5"/>
  <param name="ym_coord"  type="double" value="-1"/>
  <param name="yp_coord"  type="double" value="1.5"/>
</node>

<node respawn="false" pkg="jaco2_calibration_utils" type="record_calibration_data" name="dyn_calib_record" output="screen">
    <param name="acc_data_path" type="string" value="$(find jaco2_calibration_utils)/data"/>
    <param name="dyn_data_path" type="string" value="$(find jaco2_calibration_utils)/data"/>
</node>

<node respawn="false" pkg="jaco2_calibration_utils" type="jaco2_create_calib_data_node" name="dyn_calib_create_data">
    <param name="iterations" type="int" value="8"/>
</node>
</launch>
